packages/ffe-cards/less/icon-card.less

```diff
@@ -4,70 +4,79 @@
 .ffe-icon-card {
     .common-card-styling();
 
-    display: flex;
-    padding: @ffe-spacing-md;
-    max-width: 440px;
+    --color: var(--ffe-farge-fjell);
```

Comment:
Denne typen variabler bør helst ligge i `theme.less`, med mer konkrete navn for lesbarhetens skyld. Denne kan f.eks hete `--ffe-v-cards-stippled-border-color`. Dark mode-varianten kan legges til i media querien som ligger i samme fil.

Reply:
Skall vare i orden nå tror jag